src/components/Map/GoogleMapComponent.module.scss

```diff
@@ -0,0 +1,17 @@
+.googleMapContainer {
+  width: 80%;
+  height: 480px;
+  max-width: 540px;
+}
+
+@media screen and (min-width: 320px) {
```

Comment:
Would it be possible to use any of the defined breakpoints from the Sass theme? If that doesn't work, it might be a good idea to place the ones defined by you there. You could also probably skip the first media query and declare `min-width: 310px;` in the initial declaration for `.googleMapContainer`.